frontend/packages/knative-plugin/src/utils/icons.tsx

```diff
@@ -1,9 +1,23 @@
 import * as React from 'react';
-import { SignInAltIcon, SignOutAltIcon } from '@patternfly/react-icons';
 import * as channelIcon from '../imgs/channel.svg';
+import * as eventSinkIcon from '../imgs/event-sink.svg';
+import * as eventSourceIcon from '../imgs/event-source.svg';
 
-export const eventSourceIcon = <SignOutAltIcon />;
+const eventIconStyle = {
+  width: '1em',
+  height: '1em',
+};
 
```

Comment:
It looks like this width and height breaks the icons mentioned above (icon on the left side of a name)

```suggestion
```

See:

![image](https://user-images.githubusercontent.com/139310/174674607-338fc0ca-2fbf-4d7f-b195-dcb1ac042170.png)

vs:

![image](https://user-images.githubusercontent.com/139310/174674654-361f1913-521d-48d7-b475-8737a3feb81d.png)

Reply:
@invincibleJai here are two icons where I moved the inner circle-center to the center of the image. PTAL. :)

![event-sink](https://user-images.githubusercontent.com/139310/174674782-131ba6c6-64e3-4161-8a88-e30ae3c58cce.svg)

![event-source](https://user-images.githubusercontent.com/139310/174674784-0c99f799-80ff-4928-8348-19a0b18616ed.svg)

Reply:
Thanks @jerolimov, have updated it PTAL

<img width="1790" alt="image" src="https://user-images.githubusercontent.com/5129024/174735652-2f7632a9-8035-4e82-97b1-c0d31ef8fe36.png">

<img width="1495" alt="image" src="https://user-images.githubusercontent.com/5129024/174735743-9e211a13-ca6e-4584-8616-80f59f83fb6c.png">